tests/sqlparser_common.rs

```diff
@@ -1806,11 +1806,34 @@ fn parse_join_nesting() {
         vec![join(nest!(nest!(nest!(table("b"), table("c")))))]
     );
 
-    let res = parse_sql_statements("SELECT * FROM (a NATURAL JOIN (b))");
+
+    let sql = "SELECT * FROM a NATURAL JOIN (((b NATURAL JOIN c)))";
```

Comment:
This appears to duplicate the test above, no?

Reply:
yep , my mistake - removed